src/components/functionselect/functionselect.js

```diff
@@ -45,6 +45,15 @@ angular.module('vlui')
           ]
         };
 
+        scope.cardinalityFilter = function(timeUnit) {
```

Comment:
Hmm we're not using it in the scope, so I don't think we should expose this to the scope.